### PR TITLE
Chore/cj debug settings

### PR DESCRIPTION
**Pull request by szymonlesisz**
## Commits description

1. moving all coinjoin related debug settings from suite reducer to coinjoin reducer

2. db save and load coinjoinDebugSettings
> new db version + migtation

3. add `Testnet` coordinator setups: public/staging 
> Public = wasabi, Staging = trezor

Screenshot from 2023-01-27 10-52-54

4. location.reload unification

### CODE REVIEW

File: packages/suite/src/actions/wallet/coinjoinClientActions.ts [hunk 1]
```diff
@@ -105,7 +105,14 @@ const clientSessionPhase = (payload: CoinjoinClientEvents['session-phase']) =>
         payload,
     } as const);
 
+export const setDebugMode = (payload: CoinjoinDebugSettings) =>
```
Comment (matejkriz) [inline packages/suite/src/actions/wallet/coinjoinClientActions.ts]: nit-naming: what about setDebugSettings or setCoinjoinDebugSettings instead? setDebugMode sounds quite confusing, but on the other hand it's consistent with the suiteActions.setDebugMode... so probably leave it for now and refactor later on all places

Comment (szymonlesisz) [reply packages/suite/src/actions/wallet/coinjoinClientActions.ts]: renamed

**Comment by szymonlesisz** [comment]
resolved conflicts, renamed `coinjoinClientActions.setDebugSettings` and const `COINJOIN.SET_DEBUG_SETTINGS`, is that naming ok?

**Comment by matejkriz** [comment]
> resolved conflicts, renamed `coinjoinClientActions.setDebugSettings` and const `COINJOIN.SET_DEBUG_SETTINGS`, is that naming ok?

nice, thanks!

**Review by matejkriz — APPROVED** [review]
super nice 🎉

**Pull request merged**